plugins/core/src/automation-actions.ts

```diff
@@ -81,6 +81,10 @@ addAction(ScryptedInterface.Notifier, {
         type: 'interface',
         deviceFilter: `deviceInterface === '${ScryptedInterface.VideoCamera}' || deviceInterface === '${ScryptedInterface.Camera}'`,
     },
+    notificationChannel: {
```

Comment:
I don't think this should be in the automation ui, since it will almost certainly never be used. It can remain accessible from an automation script via the API.

Reply:
https://github.com/koush/scrypted/pull/1625/commits/564283e14f78492b97f5986a5cbffb63f63c6323